gym/envs/box2d/lunar_lander.py

```diff
@@ -618,6 +651,11 @@ def heuristic(env, s):
 
 
 def demo_heuristic_lander(env, seed=None, render=False):
+
+    # wind power must be reduced for heuristic landing
+    global WIND_POWER
```

Comment:
This is a big no-no, all caps suggests that this should be a global constant, so changing this in the code is a bad pattern.

If the wind power is something that users are meant to change, maybe make it a constructor argument?

Reply:
Hi Ariel, the wind power isn't something that user's are meant to change.

The only reason it's here is that the heuristic landing won't work with the default wind setting, and this causes it to fail tests. So for heuristic landings, I've reduced the wind power to be 1/15th of the original magnitude (so that the wind code is still tested in tests, but still solvable with heuristic landing.

I'm aware this is not the most optimal solution, I'll think of something soon.

Reply:
I've made it a kwarg in the latest commit.

Reply:
@jkterry1 done!